libs/sdk/e2e-schematics/src/utils/angular-module-generator.ts

```diff
@@ -9,3 +10,13 @@ export async function angularModuleGenerator(tree: Tree, schema: ModuleSchema) {
     schema
   );
 }
+
+export async function angularComponentGenerator(
```

Comment:
John helped with this change - long story short is that the NX component generator was giving odd import paths in the tests for `find-module` and so this function was added to be able to use Angular's component generator for the test.